### ISSUE TITLE
DirbScan Module issue

### ISSUE THREAD

**Issue by uBadRequest** [j3ssie/Osmedeus#89]
I believe the DirbScan Module is not replacing %EXT% and is just appending it to the end of the directory. Would be nice if I could skip this module now because its getting nothing but 400 errors. I believe I can fix this myself, but I assume everyone else is also having this problem.

```
[*] Waiting for DirbScan module
+ http://redacted.com/Searchadminbox.%EXT%.php (CODE:400|SIZE:242)
+ http://redacted.com/SiteServer/Admin/commerce/foundation/driver.%EXT%.php (CODE:400|SIZE:242)
+ http://redacted.com/SiteServer/admin/findvserver.%EXT%.php (CODE:400|SIZE:242)
+ http://redacted.com/Sites/Knowledge/Membership/Inspiredtutorial/Viewcode.%EXT%.php (CODE:400|SIZE:166)
+ http://redacted.com/about.%EXT%.jsp (CODE:400|SIZE:166)
+ http://redacted.com/Sites/Samples/Knowledge/Push/ViewCode.%EXT%.php (CODE:400|SIZE:242)
+ http://redacted.com/aboutus.%EXT%.jsp (CODE:400|SIZE:242)
+ http://redacted.com/access_admin.%EXT%.jsp (CODE:400|SIZE:166)
+ http://redacted.com/account.%EXT%.jsp (CODE:400|SIZE:242)
+ http://redacted.com/actions_admin.%EXT%.jsp (CODE:400|SIZE:242)
+ http://redacted.com/addadmin.%EXT%.jsp (CODE:400|SIZE:166)
```

**Comment by uBadRequest** [comment]
@j3ssie Nice tool by the way! I love it!

**Comment by j3ssie** [comment]
Thank you for the feedback!
Will be fix with #88
